src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml

```diff
@@ -4494,78 +4470,6 @@ NewApplication                                                61b5d8ea-fdc6-40a2
 			</dev:type>
 			<dev:defaultValue></dev:defaultValue>
 		</command:parameter>
-		<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="true (ByPropertyName)" position="named">
```

Comment:
TemplateParameter, TemplateFile and TemplateUri still exists, right @ravbhatnagar ?